website/docs/guides/using-terraform-templates.mdx

```diff
@@ -92,3 +95,202 @@ gitops get template terraform tf-controller-aurora --list-parameters --endpoint
+  engine                  = local.engine
+  engine_version          = local.engine_version
+  port                    = local.port
+  availability_zones      = slice(data.aws_availability_zones.available.names, 0, 3)
+  database_name           = var.database_name
+  master_username         = var.master_username
+  master_password         = var.master_password
+  backup_retention_period = var.backup_retention_period
+  skip_final_snapshot     = true
+  apply_immediately       = true
+}
+
+resource "aws_rds_cluster_instance" "cluster_instance" {
+  count              = 1
+  identifier         = "${aws_rds_cluster.mycluster.id}-${count.index}"
+  cluster_identifier = aws_rds_cluster.mycluster.id
+  instance_class     = "db.t3.small"
+  engine             = aws_rds_cluster.mycluster.engine
+  engine_version     = aws_rds_cluster.mycluster.engine_version
+}
+```
+
+Add the following template to `./clusters/<cluster-name>/rds-template.yaml` to your Git repository. 
+
+```yaml title="./clusters/<cluster-name>/rds-template.yaml"
+---
+apiVersion: clustertemplates.weave.works/v1alpha1
+kind: GitOpsTemplate
+metadata:
+  name: rds-template
+  namespace: default
+spec:
+  description: This is a sample Aurora RDS template.
+  params:
+    - name: RESOURCE_NAME
+      description: Resource Name
+    - name: CLUSTER_IDENTIFIER
+      description: Cluster Identifier
+    - name: DATABASE_NAME
+      description: Database Name
+    - name: BACKUP_RETENTION_PERIOD
+      description: Backup Retention Period
+    - name: REGION
+      description: Region
+  resourcetemplates:
+    - apiVersion: infra.contrib.fluxcd.io/v1alpha1
+      kind: Terraform
+      metadata:
+        name: ${RESOURCE_NAME}
+        namespace: flux-system
+      spec:
+        interval: 1h
+        path: ./
+        approvePlan: auto
+        alwaysCleanupRunnerPod: true
+        vars:
+        - name: cluster_identifier
+          value: ${CLUSTER_IDENTIFIER}
+        - name: database_name
+          value: ${DATABASE_NAME}
+        - name: backup_retention_period
+          value: ${BACKUP_RETENTION_PERIOD}
+        - name: region
+          value: ${REGION}
+        varsFrom:
+        - kind: Secret
+          name: tf-controller-auth
+        sourceRef:
+          kind: GitRepository
+          name: flux-system
+          namespace: flux-system
+```
+
+:::tip
+You can change the location where you keep your Terraform manifests in your Git source (which the TF-Controller will reconcile) by configuring `spec.resourcetemplates.spec.path`.
+:::
+
+Commit and push your changes.
+
+### 5. Create the TF resource
+```bash
+gitops add terraform --from-template rds-template \
+--username=<username> --password=<password> \
+--endpoint https://localhost:8000 \
+--url <config-repo> \
+--set "RESOURCE_NAME"="tf-controller-aurora","CLUSTER_IDENTIFIER"="super-awesome-aurora","DATABASE_NAME"="db1","BACKUP_RETENTION_PERIOD"=5,"REGION"="us-west-2"
+```
+
```

Comment:
It'll be awesome to show the output of this command and possibly a way to check if your manifest is reconciled correctly by TF Controller.

Reply:
Thanks! I added the output of the command and some additional instructions :)